packages/diagrams/lib/src/media_query.dart

```diff
@@ -0,0 +1,142 @@
+          primarySwatch: Colors.blue,
+        ),
+        child: new Material(
+          color: const Color(0xFFFFFFFF),
+          child: new MediaQuery(
+            data: const MediaQueryData(
+              padding: EdgeInsets.all(0.0),
+            ),
+            child: Center(
+              child: new Stack(
+                alignment: Alignment.topCenter,
+                children: <Widget>[
+                  new Container(
+                    width: 350.0,
+                    height: 250.0,
+                    color: Colors.black,
+                  ),
+                  new Container(
+                    width: 342.0,
+                    height: 246.0,
+                    color: Colors.red,
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(bottom: 130.0),
+                    child: new Container(
+                      width: 342.0,
+                      height: 246.0,
+                      color: Colors.amber,
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(bottom: 200.0),
+                    child: new Container(
+                      width: 342.0,
+                      height: 246.0,
+                      color: Colors.white,
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.fromLTRB(0.0, 170, 240.0, 0.0),
+                    child: new Container(
+                      width: 6.0,
+                      height: 80.0,
+                      color: Colors.black45,
+                    ),
+                  ),
+                  const Padding(
+                    padding: EdgeInsets.fromLTRB(0.0, 260, 240.0, 0.0),
+                    child: Text(
+                      'viewInsets',
+                      style: TextStyle(fontSize: 20.0),
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.fromLTRB(0.0, 100, 0.0, 0.0),
+                    child: Container(
+                      width: 6.0,
+                      height: 70.0,
+                      color: Colors.black45,
+                    ),
+                  ),
+                  const Padding(
+                    padding: EdgeInsets.fromLTRB(0.0, 70, 0.0, 0.0),
+                    child: Text(
+                      'padding',
+                      style: TextStyle(fontSize: 20.0),
+                    ),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.fromLTRB(240.0, 100.0, 0.0, .0),
+                    child: new Container(
+                      width: 6.0,
+                      height: 150.0,
+                      color: Colors.black45,
+                    ),
+                  ),
+                  const Padding(
+                    padding: EdgeInsets.fromLTRB(240.0, 260.0, 0.0, 0.0),
+                    child: Text(
```

Comment:
For these, I think there's a Label widget or something we can use that might look clearer.

Reply:
Using the label will draw a line directly to the text, which would go outside the bounds on the actual area I want to specify. Do you think that would be ok, or more confusing?